### PR TITLE
feat(security): Create hitrust doc template

### PR DESCRIPTION
**Pull request by bradleycamacho**

### CODE REVIEW

File: src/content/docs/security/security-privacy/compliance/hitrust.mdx [hunk 1]
```diff
@@ -0,0 +1,180 @@
+---
+title: HITRUST CSF
+tags:
+  - Security
+  - Security and Privacy
+  - Compliance
+
+metaDescription: HIPAA enablement at New Relic and requirements for HIPAA-enabled acccounts.
+---
+HITRUST Alliance is a not-for-profit organization whose mission is to champion programs that safeguard sensitive information and manage information risk for organizations across all industries and throughout the third-party supply chain. Additional information can be found on the [HITRUST webpage](https://hitrustalliance.net).
```
Comment (x8a) [inline src/content/docs/security/security-privacy/compliance/hitrust.mdx:10]: non-profit

File: src/content/docs/security/security-privacy/compliance/hitrust.mdx [hunk 2]
```diff
@@ -0,0 +1,180 @@
+---
+title: HITRUST CSF
+tags:
+  - Security
+  - Security and Privacy
+  - Compliance
+
+metaDescription: HIPAA enablement at New Relic and requirements for HIPAA-enabled acccounts.
+---
```
Comment (x8a) [inline src/content/docs/security/security-privacy/compliance/hitrust.mdx:9]: I'd add a line break between the front matter and the content.

File: src/content/docs/security/security-privacy/compliance/hitrust.mdx [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+HITRUST Alliance is a not-for-profit organization whose mission is to champion programs that safeguard sensitive information and manage information risk for organizations across all industries and throughout the third-party supply chain. Additional information can be found on the [HITRUST webpage](https://hitrustalliance.net).
+
+## Applicable document by service [#applicable-services]
+
+<Callout variant="caution">
+Not all [New Relic One](/docs/new-relic-one/use-new-relic-one/get-started/introduction-new-relic-one/) services are in compliance with this program. For non-compliant services, please see the section of [services not in scope](#not-scope).
```
Comment (x8a) [inline src/content/docs/security/security-privacy/compliance/hitrust.mdx:15]: I don't think we need the `please` in the sentence.

File: src/content/docs/security/security-privacy/compliance/hitrust.mdx [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+      <td>
+        New Relic One Platform
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+## Services not in scope [#not-scope]
+
+The following services are not HITRUST certified:
```
Comment (x8a) [inline src/content/docs/security/security-privacy/compliance/hitrust.mdx:62]: HITRSUT doesn't provide certificate for the following services:

**Comment by x8a** [comment]
Hey, @bradleycamacho! Please check the comments!

**Review by barbnewrelic — APPROVED** [review]
SME OKed via slack

**Pull request merged**